charts/admission-controller/values.yaml

```diff
@@ -12,6 +12,8 @@ global:
     secureAPITokenSecret: ""
     # The global Sysdig Secure region.
     region: "us1"
+    # The global Access Key to access Sysdig Secure.
+    accessKey: ""
```

Comment:
Isn't it the same as `secureAPIToken`?

Reply:
See comment above. This one is for the global override, while the one in line 80 is the local one.